src/room.ts

```diff
@@ -203,6 +206,9 @@ export class Room extends EventEmitter implements Sayable {
       log.warn('Room', 'parse() on a empty rawObj?')
       return null
     }
+    if (!rawObj.MemberList) {
+      rawObj.MemberList = []
```

Comment:
Delete them.

As I mentioned before, you should NOT change any value of `rawObj`.